integration/spark-datasource/src/test/scala/org/apache/spark/sql/carbondata/datasource/TestCreateTableUsingSparkCarbonFileFormat.scala

```diff
@@ -322,6 +328,56 @@ class TestCreateTableUsingSparkCarbonFileFormat extends FunSuite with BeforeAndA
     assert(new File(filePath).exists())
     cleanTestData()
   }
+  test("Read data having multi blocklet ") {
+    buildTestDataMuliBlockLet(700000)
+    assert(new File(writerPath).exists())
+    spark.sql("DROP TABLE IF EXISTS sdkOutputTable")
+
+    if (SparkUtil.isSparkVersionEqualTo("2.1")) {
+      //data source file format
+      spark.sql(s"""CREATE TABLE sdkOutputTable USING carbon OPTIONS (PATH '$writerPath') """)
+    } else {
+      //data source file format
+      spark.sql(
+        s"""CREATE TABLE sdkOutputTable USING carbon LOCATION
+           |'$writerPath' """.stripMargin)
+    }
+    spark.sql("select count(*) from sdkOutputTable").show(false)
+    val result=checkAnswer(spark.sql("select count(*) from sdkOutputTable"),Seq(Row(700001)))
```

Comment:
it says in line#331 that the number is 700000, why is the result 700001 here?
You'd better use 'until' in line#372 to keep them consistent.